src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -10,34 +10,41 @@
 public class CommandResult {
 
     private final String feedbackToUser;
+    private String tabType;
 
     /** Help information should be shown to the user. */
     private final boolean showHelp;
 
+
     /** The application should exit. */
     private final boolean exit;
 
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit, String tabType) {
         this.feedbackToUser = requireNonNull(feedbackToUser);
         this.showHelp = showHelp;
         this.exit = exit;
+        this.tabType = tabType;
     }
 
     /**
      * Constructs a {@code CommandResult} with the specified {@code feedbackToUser},
      * and other fields set to their default value.
      */
     public CommandResult(String feedbackToUser) {
-        this(feedbackToUser, false, false);
+        this(feedbackToUser, false, false, "default");
```

Comment:
Can add an overloading constructor. Then under the original constructor, assign a DEFAULT_TAB to the tabType

Reply:
resolved